show/muxcable.py

```diff
@@ -412,3 +427,97 @@ def cableinfo(port):
 
     body = [[vendor, part_num]]
     click.echo(tabulate(body, headers=headers))
+
+
+@muxcable.group(cls=clicommon.AbbreviationGroup)
+def hwmode():
+    """start configuring cable in hardware mode on a port"""
+    pass
+
+
+@hwmode.command()
+@click.argument('port', metavar='<port_name>', required=False, default=None)
+def mux_direction(port):
+    if port is not None:
+
+        """Show muxcable summary information"""
+        if platform_sfputil is not None:
+            physical_port_list = platform_sfputil_helper.logical_port_name_to_physical_port_list(port)
+
+        if not isinstance(physical_port_list, list):
+            click.echo("ERR: Unable to get a port on muxcable port")
+            sys.exit(EXIT_FAIL)
+            if len(physical_port_list) != 1:
+                click.echo("ERR: Unable to get a single port on muxcable")
+                sys.exit(EXIT_FAIL)
+
+        physical_port = physical_port_list[0]
+        import sonic_y_cable.y_cable
```

Comment:
Why import inside a method and not at the top?

Reply:
importing at top might be redundant, because some commands
like show mux status/config do not need sonic_y_cable package.
Thats why prefer to have it inside the command.

Reply:
>importing at top might be redundant

I think "unnecessary" is more correct than "redundant" here. However, I do prefer importing at the top whenever possible. For now we can leave this as-is and revisit after the y_cable lib refactor.